src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6322,12 +6561,190 @@ private IOperation VisitNoneOperationExpression(IOperation operation)
 
         public override IOperation? VisitInterpolatedStringHandlerCreation(IInterpolatedStringHandlerCreationOperation operation, int? captureIdForResult)
```

Comment:
>IInterpolatedStringHandlerCreationOperation [](http://example.com/codeflow?start=75&length=43)

Is it possible that this node is a child of an operation None and we fail to set up the placeholder replacements for it?

Reply:
I don't believe so: the only time we should have placeholders for the receiver/arguments should be an actual invocation of a method or property, and I don't believe we have any NoneOperations that would hit it today (at least, none that I see in the catch-all for `CSharpOperationFactory.Visit`).